aiida_quantumespresso/tools/pwinputparser.py

```diff
@@ -59,7 +61,7 @@ def get_kpointsdata(self):
         return kpoints
 
 
-def create_builder_from_file(input_folder, input_file_name, code, metadata, pseudo_folder_path=None, use_first=False):
+def create_builder_from_file(input_folder, input_file_name, code, metadata, pseudo_folder_path=None):
```

Comment:
This function works just fine, and might be handy in some cases. However, now it's a little hard to find (I wasn't aware of it). Would it be a good idea to implement a `.from_folder()` method for each process class that replicates this behaviour?

Reply:
Feel free to open a feature request. At some point we would need something similar anyway in order to implement immigrator functionality. I still have an open AEP for this with an implementation in `aiida-core` which I need to get back to at some point, but it would still require the functionality that constructs a builder from a folder of input files. I won't address this in this PR of course.